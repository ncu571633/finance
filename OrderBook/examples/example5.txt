# Example orders file

SELL GFD 1000 10 order1
PRINT