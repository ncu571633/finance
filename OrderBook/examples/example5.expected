SELL:
1000 10
BUY:
